tests/attackgraph/test_attackgraph.py

```diff
@@ -759,7 +759,7 @@ def test_attackgraph_transitive_advanced():
 
     test_lang_graph = LanguageGraph(MalCompiler().compile(
         'tests/testdata/transitive_advanced.mal'))
-    test_lang_graph.save_to_file('tmp/trans_adv_lang_graph.yml')
+    test_lang_graph.save_to_file('logs/trans_adv_lang_graph.yml')
```

Comment:
Why was this changed? Was it just for testing and it was left over?

Reply:
Ok, I saw that this might be related to renaming the `tmp` folder to `logs` which is a great idea. But either all of the tests should save it to `logs` , or more likely this should have been saved in `/tmp/` instead of `tmp`. I'll have a look at it and fix this.